sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonPlugin.java

```diff
@@ -91,6 +93,22 @@ public void define(Context context) {
       PythonRuleRepository.class,
       AnalysisWarningsWrapper.class);
 
+    context.addExtensions(
+      PropertyDefinition.builder(IPYNB_FILE_SUFFIXES_KEY)
+        .index(12)
+        .name("IPython File Suffixes")
+        .description("List of suffixes of IPython Notebooks files to analyze.")
+        .multiValues(true)
+        .category(PYTHON_CATEGORY)
+        .subCategory(GENERAL)
+        .onQualifiers(Qualifiers.PROJECT)
+        .defaultValue("ipynb")
+        .build(),
+      IPynb.class,
+      IPynbProfile.class,
+      IPynbSensor.class,
+      IPynbRuleRepository.class);
```

Comment:
I don't know if we should have that property, is the goal to have other extensions or to allow disabling the notebooks ?

Reply:
No, you are right I don't think we need this. I think it was to mirror the Python file properties.